temba/contacts/templatetags/contacts.py

```diff
@@ -199,7 +199,7 @@ def history_class(item):
     if isinstance(item, Msg):
         if item.media and item.media[:6] == 'video:':
             css = '%s %s' % (css, 'video')
-        if item.direction or item.recipient_count:
+        if item.direction or (item.broadcast and item.broadcast.recipient_count > 1):
```

Comment:
@smn I mean make sure that messages being displayed in the contact history have broadcast prefetched so this line doesn't turn into a db hit for each message. But am still confused about how item.direction can be non-truthy....